packages/md2mld/md2mld.0.3.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+name: "md2mld"
+synopsis: "Md files into odoc mld files"
+description: """
+`md2mld` converts a Markdown-format file into the `mld` format used by [odoc](https://github.com/ocaml/odoc) to render HTML documentation or OCaml libraries.  You can use this script to automatically embed a `README.md` file into API documentation for an OCaml  library.
+
+You can use it manually as follows
+
+```
+$ md2mld filename.md > outfile.mld
+```
+
+In `dune` you can use it to generate an mld file with
+
+```
+(rule (with-stdout-to outfile.mld (run md2mld filename.md)))
+```
+
+You can see the documentation generated from the latest tagged version of this README at [mseri.github.io/md2mld/md2mld/index.html](http://mseri.github.io/md2mld/md2mld/index.html)."""
+maintainer: "Marcello Seri <marcello.seri@gmail.com>"
+authors: "Marcello Seri <marcello.seri@gmail.com>"
+license: "ISC"
+homepage: "https://github.com/mseri/md2mld"
+doc: "https://mseri.github.io/md2mld/"
+bug-reports: "https://github.com/mseri/md2mld/issues"
+depends: [
+  "ocaml"
+  "dune" {build & >= "1.4.0"}
+  "base-bytes"
+  "omd"
+]
+flags: light-uninstall
```

Comment:
there is no remove instruction, this is unnecessary. Could you fix it upsteam as well? (I'll fix it here right away)